account_operating_unit/models/account_move.py

```diff
@@ -241,19 +211,16 @@ def _post(self, soft=True):
                     move, ou_id, ou_balances
                 )
                 if line_data:
-                    amls.append(ml_obj.with_context(wip=True).create(line_data))
+                    line_datas.append(line_data)
+            if line_datas:
+                amls = ml_obj.with_context(check_move_validity=False).create(line_datas)
             if amls:
-                move.with_context(wip=False).write(
+                move.with_context(check_move_validity=True).write(
```

Comment:
Is `check_move_validity` a new key you just created or already existing somewhere ?
Asking because if it's a new one, it isn't read anywhere, so I don't understand its purpose

Reply:
It's a key added in Odoo Community a few version ago (https://github.com/odoo/odoo/blob/96223d322c0f59b35ccda01c82e99a1ac6e2fe0c/addons/account/models/account_move.py#L1890). 
The purpose is to avoid the check on the balance.

Reply:
Alright; thank you :)